### PR TITLE
Docs(public-process): Start process from a public form

### PR DESCRIPTION
**Pull request by marcosgvieira**
Closes https://github.com/camunda/tasklist/issues/2873

### Tasklist
- Created a new page to group information on how to start processes from Tasklist. So I removed the information about processes tab from the "Overview and Example use Cases" and added it to the new page called "Starting Processes"
- Added information about how to start a process from a form in Tasklist

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [x] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/modeler/web-modeler/start-public-process.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+id: start-public-process
+title: Start processes via public form
+description: "Share executable processes easily with anyone outside publicly from the Web Modeler on Camunda SaaS."
+---
+
+<span class="badge badge--cloud">Camunda Platform 8 only</span>
+
+The Camunda Platform now offers a convenient solution for process developers who want to make their processes accessible to users without requiring them to sign up on the platform or understand the task workflow.
```
[the remark this answers is not in the document]
Comment (marcellobarile) [reply docs/components/modeler/web-modeler/start-public-process.md:9]: thanks :) Sure! Let's change it

File: docs/components/modeler/web-modeler/start-public-process.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+With this feature, developers can easily publish a process with a form assigned to the start event and enable a public link that can be shared with anyone.
```
[the remark this answers is not in the document]
Comment (marcellobarile) [reply docs/components/modeler/web-modeler/start-public-process.md:11]: yep sounds better :P

**Comment by akeller** [comment]
>  This change or page is part of a marketing blog, conference talk, or something else on a schedule.

What's the date we need to plan for this to be merged and released? I see it's still in draft.

**Comment by marcellobarile** [comment]
@marcosgvieira I pushed a commit that applied prettier's fixes

File: docs/components/modeler/web-modeler/img/public-link.png
Comment (akeller) [inline docs/components/modeler/web-modeler/img/public-link.png:1]: Is this a dead/stale/or fake link?

File: docs/components/modeler/web-modeler/advanced-modeling/publish-public-processes.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+<img src={EmbedStartFormImg} style={{width: 400}} alt="Embedding a start form" />
+
+You can preview the embedded form by clicking the **form icon** again.
+
+## Enable the public access
+
+To enable the public access, follow these steps:
+
+1. Open the **Publication** section in the **properties panel** (not the tab of the same name) and activate the toggle.
```
Comment (akeller) [inline docs/components/modeler/web-modeler/advanced-modeling/publish-public-processes.md:64]: > (not the tab of the same name)

What's going on here? Can we make this clearer? Is this a naming issue?

**Comment by christian-konrad** [comment]
@akeller 

> This is part of 8.3.0-alpha, correct? So not available to users of any older version? I just want to confirm we should only document this in next and not the current, 8.2, docs.

Exactly, just for `next`

File: docs/components/modeler/web-modeler/advanced-modeling/publish-public-processes.md (hunk 3 above)
Comment (christian-konrad) [reply docs/components/modeler/web-modeler/advanced-modeling/publish-public-processes.md:64]: @akeller there's a tab in the same screen with the same name, probably we should have named that better (especially for A11y reasons)! I created a Web Modeler issue for that: https://github.com/camunda/web-modeler/issues/5013

File: docs/components/modeler/web-modeler/img/public-link.png
Comment (akeller) [reply docs/components/modeler/web-modeler/img/public-link.png:1]: Oh sorry, I mean is the development environment link stale or fake? Or can people use it (and is that ok)?

**Review by akeller — APPROVED** [review]

**Pull request merged**